node/src/main/scala/com/wavesplatform/http/DebugApiRoute.scala

```diff
@@ -54,6 +54,7 @@ case class DebugApiRoute(
     peerDatabase: PeerDatabase,
     establishedConnections: ConcurrentMap[Channel, PeerInfo],
     rollbackTask: ByteStr => Task[Either[ValidationError, Seq[Block]]],
+    discardedTxsToUtx: Seq[Transaction] => Unit,
```

Comment:
Instead of having three rollback-related arguments, I suggest adding just one like
```scala
rollbackTask: (ByteStr, Boolean) => Task[Either[ValidationError, Unit]]
```
This task will perform rollback, return transactions to UTX if needed and broadcast local score to peers.